src/services/info/notice/notice-detail/NoticeDetailPage.vue

```diff
@@ -245,10 +264,20 @@ export default {
             modalState.deleteModalVisible = true;
         };
 
-        (async () => {
+        const initPage = async () => {
             state.loading = true;
             await getNoticePostData();
+            if (state.noticePostData.created_at) {
+                await getNextPostData(state.noticePostData.created_at);
+                await getPrevPostData(state.noticePostData.created_at);
+            }
             state.loading = false;
+        };
+        watch(() => props.postId, () => {
+            initPage();
+        });
+        (async () => {
+            await initPage();
```

Comment:
```suggestion
        watch(() => props.postId, () => {
            initPage();
        }, {immediate: true});
```

혹시 이렇게 하면 init 하기전에 렌더되어서 문제가 있나요? 
문제가 없다면 이렇게 하면 어떨까요?!